### ISSUE TITLE
🐛 Bug Report: User credentials is unexpectedly missing token

### ISSUE THREAD

**Issue by 2513483494** [backstage/backstage#27105]
### 📜 Description

I cloned your master branch, then configured gitup authentication locally, then started yarn dev, then created, and after selecting the template, an error occurred, "User credentials is unexpectedly missing token"

### 👍 Expected behavior

I hope create can be successful

### 👎 Actual Behavior with Screenshots

![image](https://github.com/user-attachments/assets/816682aa-a496-4690-8fa7-75f77379c989)
![image](https://github.com/user-attachments/assets/43a40b8b-92f3-478b-af61-1628a2f9a797)

### 👟 Reproduction steps

I cloned your master branch, then configured gitup authentication locally, then started yarn dev, then created, and after selecting the template, an error occurred, "User credentials is unexpectedly missing token"

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

None

**Comment by stephenglass** [comment]
I can replicate and confirm this issue, after upgrading from 1.31.3 to 1.32.0, unable to run templates with this error being shown.

**Comment by SmajlovicA** [comment]
I have same issue after upgrading to 1.32.0

**Comment by carlosrv999** [comment]
I am having the same problem with a fresh backstage installation 1.32.0. Installed using the backstage/create-app tool. Tried configuring Azure AD login and github login and same error.

**Comment by dotboris** [comment]
I've updated to [v1.32.1](https://github.com/backstage/backstage/releases/tag/v1.32.1) and that seems to have fixed the issue for me.

**Comment by Rugvip** [comment]
Yep this was caused by the bug that was fixed in 1.32.1, closing
